pom.xml

```diff
@@ -227,8 +227,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.1</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
@jongyoul  the ipfs jar I use requires jsk 1.8    I  also mentioned this in the google document linked above.